### PR TITLE
Remove gesture detection

### PR DESCRIPTION
**Pull request by ezequielc**
@Rise-Vision/apps Please review. Thanks

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc should probably remove the directive ref from here as well: https://github.com/Rise-Vision/rise-vision-apps/blob/master/web/scripts/storage-selector-app.js#L58. Thanks.

**Comment by ezequielc** [comment]
@alex-deaconu Good catch. I will remove. Thanks

File: bower.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "angular-ui-codemirror": "~0.3.0",
     "angular-vertilize": "~1.0.1",
     "angular-mocks": "1.5.0",
-    "common-header": "https://github.com/Rise-Vision/common-header.git#v1.13.1",
+    "common-header": "https://github.com/Rise-Vision/common-header.git#v1.13.1#feature/remove-gesture",
```
Comment (alex-deaconu) [inline bower.json]: @ezequielc I think this is a typo, but you're adding the branch name after the existing tag. I don't believe the branch is staged correctly in this case. Shouldn't be an issue in this case, just FYI. Thanks.

Comment (ezequielc) [inline bower.json]: @alex-deaconu I will update with the tagged version. Thanks

**Comment by alex-deaconu** [comment]
@ezequielc looks good 👍 . Thanks for fixing!

**Pull request merged**